adapters/postgres/postgres.go

```diff
@@ -353,36 +413,15 @@ func parseArray(value interface{}) string {
 }
 
 // Insert execute insert sql into a table
-func Insert(database, schema, table string, body api.Request) (jsonData []byte, err error) {
+func Insert(database, schema, table string, r *http.Request) (jsonData []byte, err error) {
```

Comment:
It's better to others functions (like UPDATE), don't receive `r *http.Request`. I prefer send the body just decoded...
```go
body := make(map[string]interface{})
if err = json.NewDecoder(r.Body).Decode(&body); err != nil {
        return
}```
This snippet in controller.